packages/bot/src/transformers/invite.ts

```diff
@@ -1,15 +1,23 @@
-import type { DiscordApplication, DiscordInviteCreate } from '@discordeno/types'
-import type { Application, Bot, User } from '../index.js'
+import type {
+  DiscordApplication,
+  DiscordInviteCreate,
+  DiscordInviteMetadata,
+  DiscordInviteStageInstance,
+  DiscordScheduledEvent,
+} from '@discordeno/types'
+import type { Application, Bot, ScheduledEvent, User } from '../index.js'
+import type { InviteStageInstance } from './stageInviteInstance.js'
 
 // eslint-disable-next-line @typescript-eslint/explicit-function-return-type
-export function transformInvite(bot: Bot, payload: DiscordInviteCreate) {
+export function transformInvite(bot: Bot, payload: DiscordInviteCreate | DiscordInviteMetadata) {
```

Comment:
This is not the right way to do it. I suggested using & here. See my previous comment